addons/l10n_ar_withholding/models/account_tax.py

```diff
@@ -7,13 +7,36 @@ class AccountTax(models.Model):
     _inherit = 'account.tax'
 
     l10n_ar_withholding_payment_type = fields.Selection(
-        [('supplier', 'Supplier'), ('customer', 'Customer')], 'Argentinean Withholding type',
-        compute="_compute_l10n_ar_withholding_payment_type", store=True, readonly=False)
-
+        [('supplier', 'Supplier'), ('customer', 'Customer')], 'Argentinean Withholding Type',
+        compute="_compute_l10n_ar_withholding_payment_type", store=True, readonly=False, help="Withholding tax for supplier or customer payments.")
+    l10n_ar_tax_type = fields.Selection(string='Argentinean Tax Type', selection=[
+        ('earnings_withholding', 'Earnings'),
+        ('earnings_withholding_scale', 'Earnings Scale'),
```

Comment:
you could make the distinction by seeing that a scale is defined or not?

Reply:
@jco-odoo if earnings_withholding_scale is selected, then l10n_ar_scale_id field is shown. l10n_ar_scale_id references to the scale

Reply:
We keet it as it is right now